libsawtooth/src/client/rest.rs

```diff
@@ -89,6 +89,42 @@ impl SawtoothClient for RestApiSawtoothClient {
             |item: Result<Batch, _>| item.map(|batch| batch.into()),
         )))
     }
+    /// Get the transaction with the given transaction_id from the current blockchain
+    fn get_transaction(
+        &self,
+        transaction_id: String,
+    ) -> Result<Option<ClientTransaction>, SawtoothClientError> {
+        let url = format!("{}/transactions/{}", &self.url, &transaction_id);
+        let request = Client::new().get(&url);
+        let response = request
+            .send()
+            .map_err(|err| SawtoothClientError::new_with_source("request failed", err.into()))?;
+
+        if response.status().is_success() {
+            let transaction: Single<Transaction> = response.json().map_err(|err| {
+                SawtoothClientError::new_with_source(
+                    "failed to deserialize response body",
+                    err.into(),
+                )
+            })?;
+            let clienttransaction: ClientTransaction = transaction.data.into();
```

Comment:
Add an underscore after client in the variable name

Reply:
Fixed